js/output/webpage/webpage-output.js

```diff
@@ -190,6 +190,8 @@ window.WebpageOutput = Backbone.View.extend({
         // Mostly borrowed from:
         // https://github.com/mozilla/thimble.webmaker.org/blob/master/locale/en_US/thimble-dialog-messages.json
         return ({
+            NO_DOCTYPE_FOUND: $._("A DOCTYPE declaration must be the first element on the page.", error),
+            HTML_NOT_ROOT_ELEMENT: $._("The <%(openTag_name)s> tag needs to be both an <html> tag and the root element on the page.", error),
```

Comment:
Isn't the tag already the first element on page? Maybe the error message could read "The root element should be an tag".